crud.py

```diff
@@ -0,0 +1,36 @@
+from sqlalchemy.orm import Session
+import models
+import schemas
+
+
+def get_all_authors(db: Session):
+    return db.query(models.Author).all()
+
+
+def get_author_by_name(db: Session, name: str):
+    return db.query(models.Author).filter(models.Author.name == name).first()
+
+
+def create_author(db: Session, author: schemas.AuthorCreate):
+    db_author = models.Author(name=author.name, bio=author.bio)
+    db.add(db_author)
+    db.commit()
+    db.refresh(db_author)
+    return db_author
+
+
+def get_all_books(db: Session):
+    return db.query(models.Book).all()
```

Comment:
The `get_all_books` function also lacks pagination. To comply with the task requirements, add pagination parameters and apply them to the query.